retirement_api/migrations/0003_remove_char_limit.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
A little nitpicky, but it might be nice to rename this migration file to reflect that it introduces a Spanish field as well.